src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/unnecessaryparentheses/InputUnnecessaryParenthesesOperatorsAndCasts.java

```diff
@@ -102,9 +108,7 @@ private void print(int arg)
 
     private int f7() {
         String f;
-
```

Comment:
Most probably the next reviewer will ask you to revert these unrelated changes.

Reply:
Oh these are because of the 120 lines rule , It would get broken if they were reverted back

Reply:
Thanks for sharing a reason.
I am ok to keep deletions, we are not extending Input with new code, but lines are growing due to more violations in file.